Restore.cmd

```diff
@@ -10,6 +10,11 @@ REM internal repositories.
 set RoslynSolution=%1
 if "%RoslynSolution%" == "" set RoslynSolution=%~dp0Roslyn.sln
 
+echo Deleting project.lock.json files
+pushd "%~dp0"
```

Comment:
Should we consider pushing `src` instead of the enlistment root here?  This code would potentially delete parts of the build output if for some reason project.lock.json files were placed there.  Seems odd but possible.